_episodes/07-thresholding.md

```diff
@@ -5,48 +5,42 @@ exercises: 65
 questions:
 - "How can we use thresholding to produce a binary image?"
 objectives:
-- "Explain what thresholding is an how it can be used."
+- "Explain what thresholding is and how it can be used."
 - "Use histograms to determine appropriate threshold values to use for the
 thresholding process."
 - "Apply simple, fixed-level binary thresholding to an image."
-- "Explain the difference between the `cv2.THRESH_BINARY` and 
-`cv2.THRESH_BINARY_INV` parameters to the `cv2.threshold()` function."
-- "Describe the shape of a binary image produced by one of the OpenCV 
-thresholding functions."
-- "Use the `cv2.merge()` function to create a color image from three binary
-color layers."
+- "Explain the difference between using the operator `>` or the operator `<` to
+threshold an image represented by a numpy array."
+- "Describe the shape of a binary image produced by thresholding via `>` or `<`."
 - "Explain when Otsu's method of adaptive thresholding is appropriate."
 - "Apply adaptive thresholding to an image using Otsu's method."
-- "Use the `cv2.countNonZero()` function to count the number of non-zero pixels
+- "Use the `np.nonzero()` function to count the number of non-zero pixels
 in an image."
 keypoints:
 - "Thresholding produces a binary image, where all pixels with intensities 
 above (or below) a threshold value are turned on, while all other pixels are
 turned off."
 - "The binary images produced by thresholding are held in two-dimensional NumPy
-arrays, since they have only one color value channel."
-- "The `cv2.merge()` function can be used to combine three single-channel image
-layers into a single, color image."
+arrays, since they have only one color value channel. They are boolean, hence they contain 
+the values 0 (off) and 1 (on)."
```

Comment:
not sure if it would be better to use `True` and `False` here, since it's boolean